schemas/json/theme.json

```diff
@@ -395,6 +395,10 @@
 										"description": "Kebab-case unique identifier for the font size preset.",
 										"type": "string"
 									},
+									"shortName": {
+										"description": "Short name of the font size preset. Gets used in the font size selector if fewer than 5 presets are defined.",
```

Comment:
> Gets used in the font size selector if fewer than 5 presets are defined.

I think this description should be a higher level `fontSize` property description.

How about explaining that values of 3 characters or less are valid for this property? This is because the user cannot understand why a shotName of 4 or more characters is not displayed unless investigating the code.

Reply:
Updated in [d6aa63a](https://github.com/WordPress/gutenberg/pull/46586/commits/d6aa63af9ece02909408968e4c40cd8205384330)